homework-solutions/js/hw-5/task-3.js

```diff
@@ -7,4 +7,27 @@ const word = 'hello';
 
 let vowelsAndConsonantsResult = '';
 
+const vowels = 'aeiou';
+
+vowelsAndConsonantsResult = word;
+vowelsAndConsonantsResult.trim();
+vowelsAndConsonantsResult.toLowerCase();
```

Comment:
эт вещи не изменяют изначальную строку, а возвращают новую, потому стоило написать:

```suggestion
vowelsAndConsonantsResult = word.trim().toLowerCase();
```